readme.md

```diff
@@ -158,3 +158,50 @@ More [examples](https://github.com/Tencent/rapidjson/tree/master/example) are av
   * [parsebyparts](https://github.com/Tencent/rapidjson/blob/master/example/parsebyparts/parsebyparts.cpp): Implements an `AsyncDocumentParser` which can parse JSON in parts, using C++11 thread.
   * [filterkey](https://github.com/Tencent/rapidjson/blob/master/example/filterkey/filterkey.cpp): A command line tool to remove all values with user-specified key.
   * [filterkeydom](https://github.com/Tencent/rapidjson/blob/master/example/filterkeydom/filterkeydom.cpp): Same tool as above, but it demonstrates how to use a generator to populate a `Document`.
+
+## Contributing
+
+RapidJson welcomes contributions, When contributing, please follow the code bellow.
```

Comment:
This sentence is not gramatically correct.

Reply:
I should correct 'bellow' to 'below'.. any other suggestions?